intermediate-javascript-calorie-counter/index150.js

```diff
@@ -0,0 +1,15 @@
+document.getElementById('calorie-form').onsubmit = calculate;
+
+function calculate(e) {
+  e.preventDefault();
+
+  const total = Array.from(document.getElementsByClassName('cal-control')).map(
+    meal => Number(meal.value)
+  );
+  //   console.log(total);
+}
+/*
+Now it's time to add up all of the elements in the `total` array.  You can either use a loop or map through all of the elements and add them to a variable.  But JavaScript provides a useful `reduce()` function that we can chain on our current expression.
```

Comment:
While you can use a loop to add everything in the `total` array to a variable, JavaScript provides the useful `reduce()` method.
Chain the `reduce()` method to the `Array.from()` expression.